_layouts/sip.html

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+---
+
+<div class="home">
+  <h1 class="page-heading">
+    SIP-{{ page.sip | xml_escape }}: {{ page.title | xml_escape }}
+    <a href="{{site.github.repository_url}}/blob/master/{{page.path}}"><svg role="img" aria-label="Source"
+        xmlns="https://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16">
+        <title>Source</title>
+        <path fill-rule="evenodd"
+          d="M9.5 3L8 4.5 11.5 8 8 11.5 9.5 13 14 8 9.5 3zm-5 0L0 8l4.5 5L6 11.5 2.5 8 6 4.5 4.5 3z" />
+      </svg></a>
+  </h1>
+  <h3>{{ page.description | xml_escape }}</h3>
+  <table>
+    <tr>
+      <th>Author</th>
+      <td>{% include authorlist.html authors=page.author %}</td>
+    </tr>
+    {% if page["discussions-to"] != undefined %}
+    <tr>
+      <th>Discussions-To</th>
+      <td>{% include discussion_links.html links=page.discussions-to %}</td>
+    </tr>
+    {% endif %}
+    <tr>
+      <th>Status</th>
+      <td>{{ page.status | xml_escape }}
+        {% if page.last-call-deadline != undefined %}
+    <tr>
+      <th>Last Call Deadline</th>
+      <td>{{ page.last-call-deadline | xml_escape }}</td>
+    </tr>
+    {% endif %}
+    </td>
+    </tr>
+    {% if page.category != undefined %}
+    <tr>
+      <th>Category</th>
+      <td>{{ page.category | xml_escape }}</td>
+    </tr>
+    {% endif %}
+    <tr>
+      <th>Created</th>
+      <td>{{ page.created | xml_escape }}</td>
+    </tr>
+    {% if page.updated != undefined %}
+    <tr>
+      <th>Updated</th>
+      <td>{{ page.updated | xml_escape }}</td>
+    </tr>
+    {% endif %}
+    {% if page.requires != undefined %}
+    <tr>
+      <th>Requires</th>
+      <td>{% include sipnums.html sips=page.requires %}</td>
+    </tr>
+    {% endif %}
+    {% if page["withdrawal-reason"] != undefined %}
+    <tr>
+      <th>Withdrawal reason</th>
+      <td>{{ page["withdrawal-reason"] | xml_escape }}</td>
+    </tr>
+    {% endif %}
+  </table>
+
+  <div class="toc">
+    <h2>Table of Contents</h2>
+    {% include toc.html html=content sanitize=true h_max=3 %}
+  </div>
+
+  {% include anchor_headings.html html=content anchorClass="anchor-link" beforeHeading=true %}
+
+  <h2>Citation</h2>
+  <p>Please cite this document as:</p>
```

Comment:
Hah, quite a big ego for us to be thinking someone will be citing Snaps

Reply:
Just included from the EIP/CAIP template. I can remove it if you prefer

Reply:
No opinion, up to you.